listening-comp/arabic-quiz-app/vector_search.py

```
@@ -0,0 +1,198 @@
+        
+        # Initialize the embedding model
+        self.model = SentenceTransformer('all-MiniLM-L6-v2')
+        
+        # Initialize ChromaDB
+        self.client = chromadb.PersistentClient(path=self.db_dir)
+        
+        # Create or get the collection
+        try:
+            self.collection = self.client.get_collection("arabic_quiz_questions")
+            print(f"Loaded existing collection with {self.collection.count()} questions")
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)